specification/billing/resource-manager/Microsoft.Billing/preview/2018-03-01-preview/billing.json

```diff
@@ -275,6 +275,68 @@
         }
       }
     },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/invoices/{invoiceName}/pricesheet/download": {
```

Comment:
This API is not modeled correctly.  Few things -
1. Name of the pricesheet is missing from this API. if pricesheets is a singleton, you can have the name hardcoded to "default. 
2. how will a GET on pricesheet/default be different from the download action
3. actions are modeled as a POST. So in the scenario, which I would like to understand, that you want to provide the download capability, it should be modeled as a POST .../pricesheets/default/download
4. ARM does not allow download kind of capabilities. If you want to provide a way for customers to download the pricesheet, you can provide a pointer to the storage account blob from where they can download the pricesheet. And if this were the case, you dont need a separate action, downloadURL can be a property on the pricesheet entity itself. 
Hope that makes sense. let me know if you have questions.

Reply:
Updated review as per our offline conversation. Please verify.